src/components/ListWraper/ListWraper.js

```diff
@@ -1,33 +1,37 @@
-import React from 'react';
+import React, { Component } from 'react';
 import ListItem from './ListItem/ListItem';
 
-const ListWrapper = ({ items }) => {
-  const { header, users } = items;
+class ListWrapper extends Component {
```

Comment:
skoro komponent nie ma żadnego state ani metod lifecyclowych to po co extendujesz Component?
wystaczy komponent funkcyjny

```
const ListWrapper = ({items: {header, users}}) => {
  return ( ...html);
} 
```

Reply:
extenduję ponieważ muszę przekazać metodę changeState głębiej do komponentu ListItem,
jak da się inaczej to wal :D

Reply:
no pewnie że sie da, do komponentu funckyjnego również mozna przekazać meto∂e przez propsy

const ListWrapper = ({onClick}) => ( <button onClick={onClick}></button>)